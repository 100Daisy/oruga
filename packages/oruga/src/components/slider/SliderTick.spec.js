import { shallowMount } from '@vue/test-utils'
import OSliderTick from '@components/slider/SliderTick'

let wrapper
const OSlider = {
    template: '<o-slider-stub></o-slider-stub>',
}

describe('OSliderTick', () => {
    beforeEach(() => {
        wrapper = shallowMount(OSliderTick, {
            parentComponent: OSlider
        })
    })

    it('is called', () => {
        expect(wrapper.name()).toBe('OSliderTick')
        expect(wrapper.isVueInstance()).toBeTruthy()
    })

    it('render correctly', () => {
        expect(wrapper.html()).toMatchSnapshot()
    })
})
