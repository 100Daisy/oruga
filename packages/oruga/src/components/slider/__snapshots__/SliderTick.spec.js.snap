// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OSliderTick render correctly 1`] = `
<div class="o-slider-tick" style="left: 0%;">
    <!---->
</div>
`;
